canonical-web-and-design/jaas-dashboard
# Update localstorage uses in components to use the new useLocalStorage hook

## Done

Update localstorage uses in components to use the new useLocalStorage hook.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Switch the `disableAnalytics` toggle in the "Settings" page and it should be in the localstorage (You can see this in the browser console under "Storage").
- Refresh the page and the settings toggle should remain switched.

## Details

Fixes #459
